index.html

```diff
@@ -5,16 +5,131 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
-
+    
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="container">
+        <header class="row"><!--Start Row 1-->
```

Comment:
I think these comments make the code a little hard to read but if they help you then 👍

Maybe remove them when you're done with the homework

Reply:
Thanks Chris, you are right there are a lot. I have removed unnecessary comments and now working on media query. I am actually thinking something but not sure how to do. my header includes a logo a navbar and contact/sign in icons. I want navbar come under the header and be in column direction. BAsically, once the screen size decreases to a phone level, the navbar is to be removed from the header (navbar's container) and becomes another container after the header. By doing that, I can (on a phone screen) show header on top, navbar elements in column and the hero image and so on...Researching about this I found that I can disable some parts of the html but this is a bit tricky. I am leaving this for later unless you have a basic solution